lib/elements/dom-module.html

```diff
@@ -78,9 +78,10 @@
      * @param {string} name Name of attribute.
      * @param {?string} old Old value of attribute.
      * @param {?string} value Current value of attribute.
+     * @param {?string} namespace Attribute namespace.
      * @return {void}
      */
-    attributeChangedCallback(name, old, value) {
+    attributeChangedCallback(name, old, value, namespace) {
```

Comment:
since dom-module isn't a mixin, why do we need to add the parameter? All the parameters should be optional from the implementation side, no?

Reply:
I don't think this is necessary, but AIUI it's slightly nicer for the JS runtime. Happy to drop or not.